### DISCUSSION TITLE
Scam website or real one?

### DISCUSSION THREAD

**Discussion by codeurimpulsif** [beancount/beancount#831] [General] [answered]
I just found this website: https://beancount.io/ related to this Github account: `stargately`.

It really look like a fintech scam, is it one of your website?

**Comment by dnicolodi** [comment, marked as answer]
The Beancount project has nothing to do with the beancount.io website or the services offered there. beancount.io offers services based on Beancount and Fava but none of the people involved in maintaining Beancount is involved.

The claims at beancount.io are definitely misleading, but I don't have any reason to believe that it is a scam. In particular, the ledger may be stored encrypted, but all operations on the ledger are done server-side, thus there must be access to the decrypted data server side, which makes the fact that the ledger is stored encrypted when at rest not very relevant.

**Comment by mariamar111** [comment]
thanks for information

**Comment by songxiaocheng** [comment]
That's crazy. Not only does that website use Beancount's name, but the screenshot on their website is clearly Fava, with a Beancount icon on it.
